x-pack/plugins/enterprise_search/public/applications/analytics/components/analytics_overview/analytics_collection_table.tsx

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+
+import { useValues } from 'kea';
+
+import { EuiBasicTable, EuiBasicTableColumn } from '@elastic/eui';
+
+import { i18n } from '@kbn/i18n';
+
+import { AnalyticsCollection } from '../../../../../common/types/analytics';
+import { generateEncodedPath } from '../../../shared/encode_path_params';
+import { KibanaLogic } from '../../../shared/kibana';
+import { EuiLinkTo } from '../../../shared/react_router_helpers';
+
+import { COLLECTION_VIEW_PATH } from '../../routes';
+
+interface AnalyticsCollectionTableProps {
+  collections: AnalyticsCollection[];
+  isLoading: boolean;
+}
+
+export const AnalyticsCollectionTable: React.FC<AnalyticsCollectionTableProps> = ({
+  collections,
+  isLoading,
+}) => {
+  const { navigateToUrl } = useValues(KibanaLogic);
+
+  const columns: Array<EuiBasicTableColumn<AnalyticsCollection>> = [
+    {
+      field: 'name',
+      name: 'Name',
+      render: (name: string) => (
+        <EuiLinkTo
+          to={generateEncodedPath(COLLECTION_VIEW_PATH, {
+            name,
+          })}
+        >
+          {name}
```

Comment:
suggestion: How long this name can be? You may want to use truncate maybe if it can be really long on a smaller screen size.

Reply:
yeh its a good suggestion! I wont apply it but will keep it in mind when we add more columns :)